contributed_definitions/NXsts.nxdl.xml

```diff
@@ -525,156 +525,315 @@ As all of the bellow are linked
 should be NXcollection not NXprocess-->
         <group name="resolution_indicators" type="NXprocess" optional="true">
             <!--Temperature 1>link to INSTR/STM/head Temperature 1 (K) (e.g. 4.92997E+0) # Temperature of STM head.-->
-            <link name="temp_stm_head" target="/entry/instrument/stm_head_temp"/>
+            <field name="stm_head_temp" type="NX_NUMBER" optional="true" units="NX_TEMPERATURE">
```

Comment:
is there a reason why "temp_stm_head" has been renamed to "stm_head_temp"? As I see, it is now consitent with the Field defined in INSTRUMENT

Reply:
To keep the name consistent in link target and concept to be linked.